keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -302,6 +303,12 @@ When an option graduates, its visibility should be moved to be controlled by the
 The introduction of these feature gates gives us the ability to move the option to beta and later stable without implying that all available options are stable.
 This approach is similliar to graduation criteria for `CPUManagerPolicyOptions` introduced [here](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2625-cpumanager-policies-thread-placement#graduation-criteria-of-options).
 
+In 1.28 this feature is being promoted to Beta. We propose following changes to TopologyManager policy options default visibility:
+
+- `TopologyManagerPolicyOptions` feature flag for enabling/disabling the entire feature will be enabled by default.
+- `TopologyManagerPolicyBetaOptions` feature flag for enabling/disabling beta options will be enabled by default.
```

Comment:
this seems a bug of the alpha implementation rather then else, because beta FGs should be enabled by default indeed! but it's good to point out explicitely